_rules/status-text-has-aria-live--mogq50.md

```diff
@@ -0,0 +1,73 @@
+---
+id: mogq50
+name: 'Status text update has `aria-live` property'
+rule_type: atomic
+description: |
+  This rule checks that any text update that meets the definition of a [status message][] has `aria-live` property.
+accessibility_requirements:
+  wcag21:4.1.3: # Status Messages (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:ARIA22: # Using role=status to present status messages
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:ARIA19: # Using ARIA role=alert or Live Regions to Identify Errors
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility Tree
+acknowledgments:
+  authors:
+    - Aron Janecki
+    - Jean-Yves Moyen
+---
+
+## Applicability
+
+This rule applies to any [HTML element][] that has a [text node][] as a [descendant][] in the [flat tree][] if:
+
+- **changed**: the `innerText` property of the element changes; and
```

Comment:
I think this will create weird interactions.

```
<body>
  <div aria-live="assertive">
    <span id="updating">Lorem Ipsum</span>
  </div>
</body>
```
So, the 3 elements match the first condition (having a text node descendant). Only the `span` matches **changed**. But then the `span` will fail the expectation for not having an `aria-live` property. But I kinda guess this is OK.